roles/swift-object/defaults/main.yml

```diff
@@ -2,4 +2,7 @@
 swift_object:
   ip: "{{ primary_ip }}"
   port: 6000
-  workers: 10
+  workers: "{{ swift.cluster.no_workers }}"
+  replicators: "{{ swift.cluster.no_replicators }}"
+  auditors: "{{ swift.cluster.no_auditors }}"
+  updaters: 2
```

Comment:
Default value for updaters is 1 (when not set).. However, object server had this set to 2 in templates.. I didn't want to change that behavior